src/Constraint/IsType.hack

```diff
@@ -29,13 +29,18 @@ class IsType {
       'null' => ($x ==> $x === null),
       'array' => ($x ==> \is_array($x)),
       'object' => ($x ==> \is_object($x)),
-      'resource' => (
-        $x ==> ($x is resource) ||
-          (
-            @\get_resource_type(
-              /* HH_FIXME[4110] closed resources fail is resource */ $x,
-            ) is string
-          )
+            'resource' => (
```

Comment:
did hackfmt do this?

Reply:
Yes, this is a multi-statement-bodied lambda.
The `{` try to get on their own line if they can.